src/components/Navigation/Navigation.jsx

```diff
@@ -95,11 +98,36 @@ const Navigation = () => {
   const anchor = isDesktopOrLaptop ? "left" : "top";
 
   return (
-    <div data-test="navigation-display">
+    <div data-test="navigation-display" style={{ marginBottom: "125px" }}>
       <React.Fragment key={anchor}>
-        <Button data-test="nav-button" onClick={toggleDrawer(anchor, true)}>
-          <MenuIcon />
-        </Button>
+        <AppBar position="absolute" data-test="app-bar-display">
+          <Toolbar>
+            <Grid container justify="space-between">
+              <Grid xs={1} item>
+                <IconButton
+                  edge="start"
+                  className={classes.menuButton}
+                  color="inherit"
+                  aria-label="menu"
+                  data-test="nav-button"
+                  onClick={toggleDrawer(anchor, true)}
+                >
+                  <MenuIcon />
+                </IconButton>
+              </Grid>
+              <Grid item xs={4} />
+            </Grid>
+            <Grid xs={2} item>
+              <Grid container justify="space-between">
+                <img
+                  src="../../src/assets/SnapIT-logo.png"
+                  alt="logo"
+                  style={{ height: "40px", marginLeft: "50px" }}
```

Comment:
🔧 Same here, and anywhere else, let's use mui styling properties.

Reply:
Addressed: commit d432723ed287368994edd0de8aae243c7d9ed62e